releasenotes/notes/fix-tcp-listener-deadlock-d441552a6e64263e.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix potential deadlock on Agent shutdown in TCP listener. 
```

We want to give a bit more context to users. Like mentioning it's about the logs Agent.

WDYT:

```suggestion
    Fix potential deadlock when shutting down an Agent with a log TCP listener.
```